examples/jenkins/pipeline/jenkinstemplate.json

```diff
@@ -37,6 +37,10 @@
       "apiVersion": "v1",
       "metadata": {
         "name": "${JENKINS_SERVICE_NAME}",
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
```

Comment:
This looks like it was an `ObjectReference` that is pointing to a logical resource, not a particular kind.  Its a failing of that API, but you care about the resource here, not the kind, right?

Kinds are data serialization formats.  Resources are objects that fulfill an intent.  For instance, the `Scale` kind can be sent to and from `replicationcontrollers/scale` and `deploymentconfigs/scale`.  You care about the resource, not its serialization format.